tests/test_db.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+import pytest
+
+from timer.db import ActivityDb
+from timer.exceptions import DuplicateTimerException
+
+
+@pytest.fixture(scope="function")
+def db():
+    return ActivityDb(db_url="sqlite:///:memory:", create_tables=True)
```

Comment:
As you can see in the session this class + injection saved a lot of hacking / not optimal code :)